### DISCUSSION TITLE
Please add *org_mapping* capabilities from SAML also to OAUTH

### DISCUSSION THREAD

**Discussion by rpasche** [grafana/grafana#60106] [General]
Reading through the documentation at https://grafana.com/docs/grafana/v9.3/setup-grafana/configure-security/configure-authentication/saml/#configure-organization-mapping, it looks like with `SAML`, we would have far more capabilities to map users (based on attributes from the IdP) into different (or multiple) organisations.

With OAUTH, there is basically no way to do the same.

I think, having the same capabilities of `SAML` within `OAUTH` would really help a lot.

Thanks.

**Comment by rajmohand** [comment]
Technically we would also like to see [this](https://github.com/grafana/grafana/pull/23661) in Grafana as we are trying to implement the multi tenant approach.

**Comment by rpasche** [comment]
@rajmohand I agree on this. Strange to read, that PR https://github.com/grafana/grafana/pull/23661 basically got denied because features for multiple orgs should get reduced....and...on the other side, exactly this feature exists when running with `SAML`. Also, looking at https://github.com/grafana/grafana/issues/24588, a lot of customers gave feedback that multi orgs is mandatory for them

**Reply by rajmohand** [reply]
yup and this has not reconsidered for 2 years now and I doubt weather we will be able to push this up !

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
